lib/checks/canMultiplyNthRoots.js

```diff
@@ -0,0 +1,19 @@
+const Node = require('../node');
+
+function canMultiplyNthRoots(node) {
+  // checks if node is a multiplication of nthRoot nodes
+  // all the terms has to have the same root node to be multiplied
+
+  if (!Node.Type.isOperator(node, '*')
+      || !(node.args.every(term => term.fn && term.fn.name === 'nthRoot'))){
```

Comment:
`NodeType.isFunction(node, 'nthRoot')` is maybe a slightly cleaner way of checking if the node is an `nthRoot`. Not super important.